sql_user_check_mysql/README.md

```diff
@@ -0,0 +1,19 @@
+# MYSQL User Check
+
+database_stats.sh script can be used to test connectivity with MYSQL database as well as fetch some stats and metadata. 
+The script supports 2 modes which can be specified with -m flag while running the script. One mode is stats mode which 
+will just print info like list of databases with their size. Count of tables, rows, column, size per schema. 
```

Comment:
Add somewhere in readme that mysql cli is mandatory and add instructions or link about how to install it.

Reply:
done